src/Microsoft.AspNet.Mvc.ModelBinding/ModelStateDictionary.cs

```diff
@@ -149,11 +187,9 @@ public void AddModelError([NotNull] string key, [NotNull] Exception exception)
         /// </summary>
         /// <param name="key">The key of the <see cref="ModelState"/> to add errors to.</param>
         /// <param name="exception">The <see cref="Exception"/> to add.</param>
-        /// <returns>True if the error was added, false if the dictionary has already recorded
-        /// at least <see cref="MaxAllowedErrors"/> number of errors.</returns>
-        /// <remarks>
-        /// This method only allows adding up to <see cref="MaxAllowedErrors"/> - 1. <see cref="MaxAllowedErrors"/>nt
-        /// invocation would result in adding a <see cref="TooManyModelErrorsException"/> to the dictionary.
+        /// <returns>
+        /// <c>True</c> if the error was added, <c>false</c> if the error was ignored.
```

Comment:
nit: might help to change first "error" to "given error".  that would slightly-more-strongly imply adding `TooManyModelErrorsException` doesn't cause the method to return `true` without bothering w/ the old `<remarks/>`.